picturae_importer.py

```diff
@@ -421,6 +492,8 @@ def create_locality_record(self):
                through create_sql_string and create_table record
                in order to add new locality record to database
         """
```

Comment:
placeholder string for locality if no locality. In order to link Geography ID to CO, need to create locality record which requires a non-empty locality field